stubs/Flask-Cors/flask_cors/decorator.pyi

```diff
@@ -0,0 +1,17 @@
+from datetime import timedelta
+from logging import Logger
+from typing import Any, Callable
+
+LOG: Logger
+
+def cross_origin(
+    origins: str | list[str],
+    methods: str | list[str],
+    expose_headers: str | list[str],
+    allow_headers: str | list[str],
+    supports_credentials: bool,
+    max_age: timedelta | int | str | None,
+    send_wildcard: bool,
+    vary_header: bool,
+    automatic_options: bool,
+) -> Callable[..., Any]: ...
```

Comment:
You can return a precise type here with ParamSpec from PEP 612.

Reply:
How can I use this w/o knowing the function?

Reply:
Something like this:

```suggestion
) -> Callable[[Callable[_P, ???]], Callable[_P, Response]]: ...
```
where `_P = ParamSpec("_P")`. It's not obvious from looking at the code what the return types of the original and the decorated functions are, so you'll have to figure that out.